person.rb

```diff
@@ -1,7 +1,11 @@
-class Person
+require_relative 'Nameable'
+
+class Person < Nameable
   attr_accessor :id, :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @age = age
     @id = generate_id
     @name = name
     @age = age
```

Comment:
- [ ] Please kindly take note that @age instance was declared twice, I think you can remove one for proper execution of your app. 👍